src/mmw/js/src/modeling/models.js

```diff
@@ -744,8 +745,20 @@ var ScenarioModel = Backbone.Model.extend({
                 fetchGisDataPromise = App.currentProject.fetchGisDataIfNeeded();
 
             self.fetchResultsPromise = fetchGisDataPromise.then(function() {
-                var promises = fetchResults();
-                return $.when(promises.startPromise, promises.pollingPromise);
+                // Ensure the model completes without modifications before trying to
```

Comment:
What happens if the initial run times out or fails? Will the second one trigger? This is probably a `.done` vs `.then` thing.